integration/walletextension/wallet_extension_test.go

```diff
@@ -350,20 +351,10 @@ func TestCanDecryptSuccessfullyAfterRestartingWalletExtension(t *testing.T) {
 	}
 }
 
-func TestCannotSubscribeOverHTTP(t *testing.T) {
```

Comment:
I left this test unchanged, but moved it down, so that the two subscription tests were grouped together.